static/sass/components/_post-content.scss

```diff
@@ -27,6 +27,6 @@
 .linked-post-title {
   &,
   #{$link} {
-    background-image: none; // no underlines on related post title links
+    text-decoration: none;
```

Comment:
I wonder if this is needed, or if we should delete this all together to make the links the same: 
![meet-the-brid-links](https://user-images.githubusercontent.com/1581694/34183480-9fc70f08-e4e0-11e7-96c4-6d609019da39.jpg)